prismic-model/json/collection-venues.json

```diff
@@ -6,18 +6,57 @@
         "label": "Title"
       }
     },
-    "description": {
+    "displayTitle": {
```

Comment:
What's the difference between `displayTitle` and `title`?
Ping @gestchild and @davidpmccormick

Reply:
title is basically the name of the collectionVenue and not used to visually display. This is existing model structure. I will change the model title to name. 

The name gets used in _app.js line 360. 

DisplayTitle is used to visually display for the end user. I will change this to title now.

Reply:
Actually speaking to @gestchild @davidpmccormick , i will keep existing model title as this will break existing contracts. I will change displayTitle to heading.